src/main/java/jenkins/plugins/office365connector/FactsBuilder.java

```diff
@@ -50,14 +50,50 @@
     final static String NAME_FAILED_TESTS = "Failed tests";
     final static String NAME_SKIPPED_TESTS = "Skipped tests";
     final static String NAME_PASSED_TESTS = "Passed tests";
+    final static String NAME_TESTS = "Tests";
```

Comment:
How it is different from line 49?

Reply:
This fact ("test") contains not total but failed, skipped and passed text together. Also I thought it is a good idea to have possibility to change text displayed in default and "compact" notifications independently.